app/templates/reading_list.html

```diff
@@ -0,0 +1,65 @@
+{% extends "base_root.html" %}
+
+{% block navlinks %}
+  {{ super() }}
+{% endblock %}
+
+{% block main_body %}
+
+<div class="container">
+<div id="readinglist">
+    <div class="col-lg-10 ml-auto mr-auto">
```

Comment:
General remark with regards to coding style: For HTML try to be consistent with regards to the indentation you use. In this template, you sometimes use 2 spaces then 3 spaces, then 4 spaces or 8 spaces.

- [ ] Please go over the file and normalize indentation.

Reply:
If you use Visual Studio Code, just select the html and hit `shift`+`alt`+`f` to auto format the code. For other text editors just ask good old google ;-)